components/camel-tahu/src/test/java/org/apache/camel/component/tahu/services/HiveMQService.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.component.tahu.services;
+
+import org.apache.camel.test.infra.common.services.TestService;
+
+public interface HiveMQService extends TestService {
```

Comment:
You can move this to an other test services to a new module in the `test-infra` dir.

Reply:
Done, including separate service suppliers for straight HiveMQ container and one built with a Dockerfile to incorporate the Sparkplug TCK extension as a new image.